tests/dummy/app/pods/docs/quickstart/template.md

```diff
@@ -1,76 +1,56 @@
-## Quickstart.
+# Quickstart.
 
-To use `Ember Table`, you need to create `columns` and `rows` dataset.
+## Installation
 
-`columns` is an array of objects which has multiple fields to define behavior of column.
-Two required field in each object is `name` and `valuePath`.
+```sh
+ember install ember-table
+```
+
+## Basic Usage
+
+To use `Ember Table`, you'll need to create column definitions, and you'll need
+to pass in an array of rows. Here is a component definition with some basic
+column definitions and a rows array:
 
 ```javascript
-  columns: [
+import Component from '@ember/component';
+
+class DemoComponent extends Component {
+  columns = [
     {
-      name: `Open time`,
-      valuePath: `open`
+      name: `First Name`,
+      valuePath: `firstName`
     },
     {
-      name: `Close time`,
-      valuePath: `close`
+      name: `Last Name`,
+      valuePath: `lastName`
     }
-  ]
-```
-
-`rows` could be a javascript array, ember array or any data structure that implements `length` and
-`objectAt(index)`. This flexibity gives application to avoid having all data at front but loads more
-data as user scrolls. Each object in the `rows` data structure should contains all fields defined
-by all `valuePath` in `columns` array.
-
-```javascript
-  rows: computed(function() {
-    const rows = emberA();
-
-    rows.pushObject({
-      open: '8AM',
-      close: '8PM'
-    });
-
-    rows.pushObject({
-      open: '11AM',
-      close: '9PM'
-    });
+  ];
 
-    return rows;
-  })
+  rows = [
+    {
+      firstName: 'Cyril',
```

Comment:
Is this a test to see if I'm doing the code review diligently? 😄 

we can find a better example 😉

Reply:
lol, I just needed a first name. If you want me to change it I can, no worries